tests/manager/api/controller/test_loan.py

```diff
@@ -131,7 +131,10 @@ class OPDSSerializationTestHelper:
             (None, OPDSFeed.ENTRY_TYPE),
             ("default-foo-bar", OPDSFeed.ENTRY_TYPE),
             (AtomFeed.ATOM_TYPE, OPDSFeed.ENTRY_TYPE),
-            (OPDS2Serializer.CONTENT_TYPE, OPDS2Serializer.CONTENT_TYPE),
+            (
+                OPDS2Serializer.CONTENT_TYPE,
+                OPDS2Serializer.CONTENT_TYPE,
+            ),
```

Comment:
```suggestion
            (OPDS2Serializer.CONTENT_TYPE, OPDS2Serializer.CONTENT_TYPE),
```

Minor: This change seems unnecessary. Its nice to have these be consistent, instead of having this one entry on multiple lines.